### PR TITLE
chore(dotnet): no nested namespace in transport channels

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: utils/doclint/generateDotnetApi.js [hunk 1]
```diff
@@ -226,11 +226,6 @@ for (const [name, type] of modelTypes)
 for (const [name, literals] of enumTypes)
   renderEnum(name, literals);
 
-if (process.argv[3] !== '--skip-format') {
```
Comment (yury-s) [inline utils/doclint/generateDotnetApi.js:229]: Why was this removed?

**Review by yury-s — APPROVED** [review]

File: utils/doclint/generateDotnetApi.js (hunk 1 above)
Comment (mxschmitt) [reply utils/doclint/generateDotnetApi.js:229]: it's done here now: https://github.com/microsoft/playwright-dotnet/blob/main/build.sh#L69
